lib/middleware/token.js

```diff
@@ -52,15 +54,15 @@ var TokenError = require('../errors/tokenerror');
  */
 module.exports = function token(server, options) {
   options = options || {};
-  
-  if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
-  
+
+  if (!server) { throw new TypeError(g.f('{{oauth2orize.token}} middleware requires a server argument')); }
```

Comment:
Of course, this means that the examples may not fit in like a square peg into a square hole, and we'll have to consider each one case by case once the translations come back.